wid/gap.py

```diff
@@ -933,11 +933,24 @@ def hdl_wid_232(desc):
     return True
 
 
+def hdl_wid_400(desc):
+    btp.set_filter_accept_list()
+    bd_addr = '000000000000'
+    bd_addr_type = btp.OwnAddrType.le_resolvable_private_address
```

Comment:
Please update doc/btp_spec.txt:
- Add documentation for Set Filter Accept List
- Update Gap Connect documentation and mention the special behavior when IUT sends zero address.

Reply:
https://github.com/intel/auto-pts/pull/640